style.css

```diff
@@ -399,31 +406,166 @@ form > * {
   color: #ebebff;
 }
 
-form > input:focus {
-  border: 2px solid #9490f5;
+.desk {
+  display: none;
+}
+
+.elementRight-1 {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  .toolbar {
+    max-height: 72px;
+    max-width: 100%;
+    justify-content: space-evenly;
+  }
+
+  .logo {
+    line-height: 20px;
+  }
+
+  .nav > .button-nav {
+    display: none;
+  }
+
+  nav > ul {
+    display: flex;
+    gap: 8px;
+  }
+
+  nav > ul > .nav-link1 {
+    padding-bottom: 8px;
+  }
+
+  .sub-container-1 {
+    background-image: url(icons&imgs/twoCirclesHeader.png);
+  }
+
+  .elementRight-1 {
+    display: flex;
+  }
+
+  .headline {
+    width: fit-content;
+  }
+
+  hr.divider-1 {
+    width: 100%;
+  }
+
+  .spprtSect {
+    max-width: 448px;
+  }
+
+  .sect {
+    min-width: 100%;
+    justify-content: space-between;
+    max-height: 215px;
+  }
+
+  .frame {
+    max-height: 215px;
+  }
+
+  .spprtItems {
+    flex-direction: row;
+    justify-content: space-between;
+    width: calc((1023 / 1440) * 100vw);
+  }
+
+  .sub-container-2 {
+    background-image: url(icons&imgs/desktopSub2BG.png);
+  }
+
+  .featuredImg {
+    flex-direction: row;
+    min-width: var(--mobile-medCont-width);
+  }
+
+  .imgPlaceHolder {
+    min-width: calc((684 / 1440) * 100vw);
+    height: auto;
+  }
+
+  .featuredImg > .one {
+    order: 1;
+  }
+
+  .featuredImg > .two {
+    order: 2;
+  }
+
+  ul.tags {
+    width: calc((311 / 1440) * 100vw);
+  }
+
+  form > textarea {
+    width: calc((660 / 1440) * 100vw);
+    height: 136px;
+  }
+
+  form > input {
+    height: 16px;
+  }
+
+  form > .field2 {
+    width: 652px;
+  }
```

Comment:
- [ ] Your inputs are responsive and that's great 🔥 💪 . In addition to what you have done, I would suggest  you to give them styles so that they have the **same width**. Look at the comparison below 👇 
**Your design**
![image](https://user-images.githubusercontent.com/90222110/177842526-913ac4ce-d973-41b1-a319-c0d40e9c7551.png)
**Figma**
![image](https://user-images.githubusercontent.com/90222110/177842643-44f76546-d6a1-4545-b411-63013d8c479a.png)

- [ ] Also as you can see in the comparison above, you need to **add more spacing** between the `textarea` input and the `send` button. In fact, you can give your send button a `margin-top` and `margin-bottom` which will give more space above it and below it 😃. What do you think?